www/src/examples/Tabs/NoAnimation.js

```diff
@@ -4,10 +4,10 @@
   id="noanim-tab-example"
   className="mb-3"
 >
-  <Tab eventKey="home" title="Home">
+  <Tab eventKey="home" title="Home" transition={false}>
```

Comment:
The changes in this file can be reverted.  The tab panes will inherit the transition prop from the parent

Reply:
Done!